<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeSet</class>
 <widget class="QDialog" name="ChangeSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="main_Layout">
     <item>
      <widget class="QWidget" name="student_widget" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="association_widget" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="button_box" native="true">
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>51</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="accept_pushButton">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancel_pushButton">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>退出</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
